tableauserverclient/server/endpoint/server_info_endpoint.py

```diff
@@ -0,0 +1,21 @@
+from .endpoint import Endpoint
+from ...models import ServerInfoItem
+import logging
+
+logger = logging.getLogger('tableau.endpoint.server_info')
+
+
+class ServerInfo(Endpoint):
+    def __init__(self, parent_srv):
+        super(Endpoint, self).__init__()
```

Comment:
This was another thing that I noticed while doing this change.  EVERY Endpoint takes parent_srv, so I'm going to rip all of the custom ctors out and just allow the base object's ctor which doesn't currently take a parent_srv take the parent_srv